graylog2-web-interface/src/components/maps/configurations/GeoIpResolverConfig.tsx

```diff
@@ -175,6 +180,17 @@ const GeoIpResolverConfig = ({ config = defaultConfig, updateConfig }: Props) =>
                                disabled={!values.enabled}
                                label="Path to the ASN database"
                                name="asn_db_path" />
+                  <TimeUnitInput label="Database refresh interval"
+                                 update={(value, unit) => {
+                                   setFieldValue('refresh_interval', value);
+                                   setFieldValue('refresh_interval_unit', unit);
+                                 }}
+                                 help="Interval at which the database files are checked for modifications and refreshed changes are detected on disk."
+                                 value={values.refresh_interval}
+                                 unit={values.refresh_interval_unit || 'MINUTES'}
+                                 defaultEnabled={values.enabled}
```

Comment:
I'm having the issue where the time selector is always disabled in the UI but adding this got it working (Disclaimer: not a UI expert)
```suggestion
                                 defaultEnabled={values.enabled}
                                 enabled={values.enabled}
```

Reply:
Thanks.  I had forgotten to test that.  `defaultEnabled` is just, well, the default. So toggling the enabled checkbox requires the other attributes.